docs/en/02_Developer_Guides/01_Templates/03_Requirements.md

```diff
@@ -179,7 +241,7 @@ is not appropriate. Normally a single backend is used for all site assets, so a
 replaced. For instance, the below will set a new set of dependencies to write to `app/javascript/combined`
 
 
-```yaml
+```yml
```

Comment:
FYI, these are synonymous in GitHub markdown rendering: https://github.com/github/linguist/blob/e761f9b013e5b61161481fcb898b59721ee40e3d/lib/linguist/languages.yml#L5480-L5484